.semaphore/semaphore.yml.d/blocks/20-typha.yml

```diff
@@ -11,6 +11,12 @@
       - name: "Typha: UT and FV tests"
         commands:
           - ../.semaphore/run-and-monitor make-ci.log make ci EXCEPT=k8sfv-test
+      - name: "Typha: UT and FV tests on UBI-minimal"
+        commands:
+          - ../.semaphore/run-and-monitor make-fv-ubi.log make clean image fv
+        env_vars:
```

Comment:
I think you can set the `CALICO_BASE` variable directly in the new pipeline rather than introducing a new `USE_UBI_AS_CALICO_BASE` flag. You can add additional checks before typha UT/FVs to make sure that the base is indeed switched.

```
env_vars:
  - name: CALICO_BASE
    value: registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Reply:
That's what I had, but I re-did it to satisfy: https://github.com/projectcalico/calico/pull/9710#discussion_r1915011861